hudi-client/src/main/java/org/apache/hudi/table/action/commit/UpsertPartitioner.java

```diff
@@ -99,7 +100,7 @@ public UpsertPartitioner(WorkloadProfile profile, JavaSparkContext jsc, HoodieTa
     assignInserts(profile, jsc);
 
     LOG.info("Total Buckets :" + totalBuckets + ", buckets info => " + bucketInfoMap + ", \n"
-        + "Partition to insert buckets => " + partitionPathToInsertBuckets + ", \n"
+        + "Partition to insert buckets => " + partitionPathToInsertBucketInfos + ", \n"
```

Comment:
did you check if this prints the value or the hash value for InsertBucketCumulativeWeightPair? if hash value, you may need to override toString for InsertBucketCumulativeWeightPair

Reply:
The parent class Pair has toString() method:
```
  @Override
  public String toString() {
    return new StringBuilder().append('(').append(getLeft()).append(',').append(getRight()).append(')').toString();
  }
```